projects/angular/src/emphasis/_labels.clarity.scss

```diff
@@ -111,27 +117,49 @@
       label-variables.$clr-label-letter-spacing,
       variables.$clr-use-custom-properties
     );
+    @include mixins.css-var(
+      line-height,
+      clr-label-line-height,
+      // line-height needed to be adjusted to force center alignment
+      label-variables.$clr-label-font-size + variables.$clr_baselineRem_1px,
+      variables.$clr-use-custom-properties
+    );
 
-    // line-height needed to be adjusted to force center alignment
-    line-height: label-variables.$clr-label-font-size + variables.$clr_baselineRem_1px;
     display: inline-flex;
     justify-content: center;
     align-items: center;
-    padding: 0 label-variables.$clr-label-padding-left-right 0;
+    @include mixins.css-var(
+      padding,
+      clr-label-padding,
+      0 label-variables.$clr-label-padding-left-right 0,
+      variables.$clr-use-custom-properties
+    );
+    @include mixins.css-var(gap, clr-label-gap-size, 0, variables.$clr-use-custom-properties);
     @include mixins.css-var(
       border-radius,
       clr-label-border-radius,
       variables.$clr_baselineRem_0_5,
       variables.$clr-use-custom-properties
     );
-    border: label-variables.$clr-label-borderwidth solid;
+    @include mixins.css-var(
+      border-width,
+      clr-label-border-width,
+      label-variables.$clr-label-borderwidth,
+      variables.$clr-use-custom-properties
+    );
+    border-style: solid;
```

Comment:
There are 2 other hard coded usages of `border: label-variables.$clr-label-borderwidth solid;` which may override the token change. Lines 67 and 20.